### ISSUE TITLE
cannot download coco 2014

### ISSUE THREAD

**Issue by breadbread1984** [tensorflow/datasets#2398]
**Short description**
I try to download coco2014 with the following code.

```python
trainset = tfds.load(name = "coco/2014", split = tfds.Split.TRAIN, download = False);
```

tfds complains that

>2020-09-07 10:38:48.913458: E tensorflow/core/platform/cloud/curl_http_request.cc:611] The transmission  of request 0x58a3e90 (URI: https://www.googleapis.com/storage/v1/b/tfds-data/o/dataset_info%2Fcoco%2F2014%2F1.1.0?fields=size%2Cgeneration%2Cupdated) has been stuck at 0 of 0 bytes for 61 seconds and will be aborted. CURL timing information: lookup time: 0.00351 (No error), connect time: 60.0547 (No error), pre-transfer time: 0 (No error), start-transfer time: 0 (No error)

I try to open the link given in the error message and get the json response

```json
{
  "error": {
    "code": 404,
    "message": "No such object: tfds-data/dataset_info/coco/2014/1.1.0",
    "errors": [
      {
        "message": "No such object: tfds-data/dataset_info/coco/2014/1.1.0",
        "domain": "global",
        "reason": "notFound"
      }
    ]
  }
}
```

by the way, how can I use the downloaded 1.0.0 version coco2014 tfds dataset?

**Environment information**
* Operating System: ubuntu 18.04
* Python version: 3.6.9
* `tfds-nightly` version: 3.2.1.dev202009060105
* `tensorflow` version: 2.3.0

**Reproduction instructions**

```
trainset = tfds.load(name = "coco/2014", split = tfds.Split.TRAIN, download = False);
```

**Link to logs**
If applicable, <link to gist with logs, stack trace>

**Expected behavior**
the coco 2014 should be download without problem.

**Additional context**

**Comment by vijayphoenix** [comment]
Could not reproduce this issue on Google Colab.
Can you try again and let us know if you still face the issue.

**Comment by breadbread1984** [comment]
i use tfds on pc not on cloud. i tried the above operation with and without tunnel. both failed. i tried the link shown in the error message and got the above 404 json reponse.

if i want to generate coco on my local disk, do i need a google cloud account or any other prerequisites？

is it possible to access tfds from China？

**Comment by vijayphoenix** [comment]
Yes, I think users from china might have to use a VPN to access the datasets.

**Comment by breadbread1984** [comment]
i use a vpn.

**Comment by Conchylicultor** [comment]
Could you test:

```py
assert tfds.core.utils.gcs_utils.exists('gs://tfds-data/dataset_info/coco/2014/1.1.0') == False
```

and report the logs, stacktrace ?

I think you could safely ignore those logs. It simply mean the pre-computed dataset info is not present on GCS, so generating the dataset will recompute the info.
